cmdutil/https/config.go

```diff
@@ -1,13 +1,17 @@
 package https
 
+import "time"
+
 // Config for HTTP and HTTPS servers.
 type Config struct {
 	InsecurePort int `env:"HEROKU_ROUTER_HTTP_PORT,required"`
 	SecurePort   int `env:"HEROKU_ROUTER_HTTPS_PORT,required"`
 
 	// These environement variables are automatically set by ACM in
 	// relation to Let's Encrypt certificates.
-	ServerCert  string `env:"SERVER_CERT,required"`
-	ServerKey   string `env:"SERVER_KEY,required"`
-	UseAutocert bool   `env:"HTTPS_USE_AUTOCERT"`
+	ServerCert   string        `env:"SERVER_CERT,required"`
+	ServerKey    string        `env:"SERVER_KEY,required"`
+	UseAutocert  bool          `env:"HTTPS_USE_AUTOCERT"`
+	ReadTimeout  time.Duration `env:"READ_TIMEOUT"`
```

Comment:
Could we perhaps group these separately from the ACM/LE-related things here? And should they have more specific names, such as HTTP_SERVER_READ_TIMEOUT?

Reply:
@danp do you have a suggestion in terms of the grouping? I stuck with ReadTimeout and Writetimeout to keep parity with the actual property names. These are TCP timeouts essentially that are applied in the context of HTTP so by naming them HTTP_SERVER_READ_TIMEOUT, would that still be accurate? Happy to take suggestions here and follow any best practices, still getting to grips with Go :)

Reply:
Note the comment above, which says, "These environement variables are automatically set by ACM in relation to Let's Encrypt certificates." That makes it seem like these configurations are also part of the ACM config. It would be clearer if you threw in a newline and a separate comment for the timeout configurations.

+1 to calling the environment variable HTTP_SERVER_READ/WRITE_TIMEOUT. ReadTimeout and WriteTimeout are fine for the struct field names since it's clear from context.

Reply:
Oh yes that totally makes sense, I'll definitely add that comment to separate the values.

+1 for calling the env vars HTTP_SERVER_READ/WRITE_TIMEOUT.